micropub.php

```diff
@@ -676,6 +669,14 @@ public static function generate_post_content( $args ) {
 			}
 		}
 
+		$checkin = $props['checkin'][0];
```

Comment:
minor nit: should this be `isset( $props['checkin'] )` instead? it was there before, i know you didn't write it, but just checking since we're here.

Reply:
I do run tests locally. I pushed it because I just couldn't see it.